### DISCUSSION TITLE
no wifi on wemos s2 mini clone board (esp32 s2)

### DISCUSSION THREAD

**Discussion by spikey1973** [micropython/micropython#10378] [ESP32] [answered]
Hey to all, 

During these days i am using a lot of my holiday time fiddling with my project based on the wemos s2 mini board. 
Currently I am trying to connect to it via webrepl. i have seen a few tutorials and have basically followed there guide but currently i am stuck on the most baics of things, I cant find the wemos board wifi connection, rather essential. 

I currently have done the following steps. 
1) flash micropython on the wemos s2 board (instead of the circuitpython i comes with) 
2) i connect with that via data cable through uPYcraft without issues. (com6) 
3) imported web_repl_setup succesfully
4) I have gotten my win11 system + wsl2 Ubuntu version read to run rshell, so i can modify booot.py though nano editor. 
5) but when I try to find the board on my laptop / mobile phone (both 2.4 and 5.0) i can't find the board at all, and no wifi no connection. 

Is there anything else I need to do so the board starts transmitting a wifi signal. 

Kind greats and best wishes for the new year

**Comment by robert-hh** [comment, marked as answer]
> Is there anything else I need to do so the board starts transmitting a wifi signal.

You have to turn wifi on. See e.g.: https://docs.micropython.org/en/latest/esp32/quickref.html#networking

**Reply by spikey1973** [reply]
thank you for pointing this out, i actually ran into this and assumed i needed this, but i couldn't find how to modify boot.py in uPycraft, the tutorial showed connection before linking with rshell, and only htne to modify boot.py. so this got me confused. I am now installong thonny and am trying to get familiar with it to modify boot.py so i can set this part up.

**Reply by spikey1973** [reply]
And.... working perfectly, Where cn I send you your cup of coffee or cold pint ;)
